spec/factories/case_log.rb

```diff
@@ -153,6 +153,7 @@
       is_carehome { "No" }
       chcharge { 7 }
       letting_in_sheltered_accomodation { "No" }
+      la_known { "Yes" }
```

Comment:
I had to add this line for the test here: `./spec/features/form/check_answers_page_spec.rb:194` to work as it relies on the completed case log. I'm not sure why only this test failed but the other tests relying on the completed case log passed when this was missing from the case log 🤔

Reply:
Probably because whether this is required or not depends on what other fields you're setting? _Might_ be safer to set this in the specific test you need it in so you can be sure you haven't accidentally stopped any other tests from legitimately failing if their functionality changes?

Reply:
I'm not setting any other fields, the test in question just use the exact ones defined by the completed case log factory because I'm less interested in the fields and more interested is testing the redirection with a completed case log so all l do in that test is use a completed case log created using the factory, no other setup involved